diff --git a/render.c b/render.c
--- a/render.c
+++ b/render.c
@@ -78,14 +78,14 @@
 			cairo_select_font_face(cairo, state->font_family,
 					       CAIRO_FONT_SLANT_NORMAL,
 					       CAIRO_FONT_WEIGHT_NORMAL);
-			cairo_set_font_size(cairo, 14);
+			cairo_set_font_size(cairo, 48);
 			set_source_u32(cairo, state->colors.border);
 			// buffer of 12 can hold selections up to 99999x99999
 			char dimensions[12];
 			snprintf(dimensions, sizeof(dimensions), "%ix%i",
 				 sel_box->width, sel_box->height);
-			cairo_move_to(cairo, sel_box->x + sel_box->width + 10,
-				      sel_box->y + sel_box->height + 20);
+			cairo_move_to(cairo, sel_box->x + sel_box->width + 24,
+				      sel_box->y + sel_box->height + 48);
 			cairo_show_text(cairo, dimensions);
 		}
 	}
}